content/reference/plugins/jenkins/index.md

```diff
@@ -95,12 +95,10 @@ If you don't have any assertions in your Gatling simulation, the JUnit step will
 
 ## Usage
 
-A new Gatling FrontLine simulation will be started every time the job is run. Check the Console Output to check the simulation progress. If the simulation ran successfully, it will look like the following:
+A new Gatling Enterprise simulation will be started every time the job is run. Check the Console Output to check the simulation progress. If the simulation ran successfully, it will look like the following:
```

Comment:
```suggestion
A new Gatling Enterprise simulation will be started every time the job is run. Check the Console Output to see the simulation progress. If the simulation ran successfully, it will look like the following:
```